docs/docs/settings/error_codes.md

```diff
@@ -52,6 +52,40 @@ See [INVE-W1](#inve-w1)
 
 See [INVE-W1](#inve-w1)
 
+#### INVE-W4
+**Server is running in debug mode - Backend**
+
+InvenTree is running in debug mode. This is **not** recommended for production use, as it exposes sensitive information and makes the server more vulnerable to attacks. Debug mode is not intended for production/exposed instances, **even for short duration**.
+
+It is recommended to run InvenTree in production mode for better security and performance. See [Debug Mode Information](../start/intro.md#debug-mode).
+
+#### INVE-W5
+**Background worker process not running - Backend**
+
+The background worker seems to not be running. This is detected by a heartbeat that runs all 5 minutes - this error triggers after not being run in the last 10 minutes.
+Check if the process for background workers is running and reaching the database. Steps vary between deployment methods.
+See [Background Worker Information](../start/processes.md#background-worker).
+
+#### INVE-W6
+**Server restart required - Backend**
+
+The server needs a restart due to changes in settings. Steps very between deployment methods.
+
+
+#### INVE-W7
+**Email settings not configured - Backend**
+
+Not all required settings for sending emails are configured. Not having an email provider configured might lead to degraded processes as password reset, update notifications and user notifications can not work. Setting up email is recommended.
+See [Email information](../start/config.md#email-settings).
+
+
+#### INVE-W8
```

Comment:
@SchrodingersGat not sure if this should be an error or warning

Reply:
I think a warning personally